manifest.json

```diff
@@ -0,0 +1,45 @@
+---
+permalink: manifest.json
```

Comment:
Enforces specific file path, if wanting explicit paths, must be set (or permalink scheme from config understood).